core/lib/multistore.ts

```diff
@@ -109,9 +109,9 @@ export function localizedRoute (routeObj: Route | string, storeCode: string) {
 }
 
 export function setupMultistoreRoutes (config, router, routes) {
-  if (config.storeViews.mapStoreUrlsFor.length > 0 && config.storeViews.multistore === true) {
+  if (config.storeViews.mapStoreUrlsFor.length > 1 && config.storeViews.multistore === true) {
```

Comment:
I think it should be > 0
```suggestion
  if (config.storeViews.mapStoreUrlsFor.length > 0  && config.storeViews.multistore === true) {
```

Reply:
... this is because the default URLs are always mapped anyway in the https://github.com/DivanteLtd/vue-storefront/blob/76a75759550ec2b7dd9642eb3b281566c1590dc5/src/themes/default/index.js#L20 - so when there is even one additional `storeCode` to map in the `storeViews.mapStoreUrlsFor` then it should be mapped

Reply:
thank you @pkarw  for explaining